CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing to Splink
+
+Contributing to an open source project takes many forms. Below are some of the ways you can contribute to Splink!
+
+## Asking questions
+
+If you have a question about Splink, we recommended asking on our GitHub [discussion board](https://github.com/moj-analytical-services/splink/discussions). This means that other users can benefit from the answers too! On that note, it is always worth checking if a similar question has been asked (and answered) before.
+
+## Reporting issues
+
+Is something broken? Or not acting how you would expect? Are we missing a feature that would make your life easier? We want to know about it!
```

Comment:
Do we want to split these out in the README too?

Reply:
https://github.com/moj-analytical-services/splink#support

Reply:
I have just changed this to signpost the contributing guide, rather than having the information in multiple places.